### PR TITLE
Improve build time

### LINKED ISSUE #1937: Build time improvements
[Issuehunt badges](https://issuehunt.io/r/sindresorhus/refined-github/issues/1937)

`npm run build` takes 6 full seconds and even in `npm run watch` it takes 2+ seconds for any change. 

It would be great to bring this time down, maybe with https://github.com/Realytics/fork-ts-checker-webpack-plugin or the new [`--incremental`](https://devblogs.microsoft.com/typescript/announcing-typescript-3-4/#faster-subsequent-builds-with-the---incremental-flag) flag, if it works

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $40.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($40.00)
### Submitted pull Requests
- [#2199 Improve build time](https://issuehunt.io/r/sindresorhus/refined-github/pull/2199)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/sindresorhus/refined-github/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by bfred-it**
Closes #1937

I disabled `stats: 'errors-only'` to have webpack say exactly how long it takes for the build.

I tried `incremental: true` in `tsconfig.json` but it had no effect. From what I understand, this currently only works when using `tsc` without extra build tools, since they haven't added any APIs for it yet.

## `npm run build`

Before | After
--- | ---
~6s | ~5s

## `npm run watch` (updates)

Before | After
--- | ---
~3.5s | ~0.5s 😍

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#1937: Build time improvements](https://issuehunt.io/repos/51769689/issues/1937)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

File: source/features/clean-issue-filters.tsx [hunk 1]
```diff
@@ -4,11 +4,11 @@ import features from '../libs/features';
 import * as api from '../libs/api';
 import {getOwnerAndRepo} from '../libs/utils';
 
-interface CacheEntry {
+type CacheEntry = {
```
Comment (notlmn) [inline source/features/clean-issue-filters.tsx]: 👍

Comment (bfred-it) [reply source/features/clean-issue-filters.tsx]: The other issue I mentioned was solved/avoided by d94acb68a515a9e1bf5cfe8241837de495996c86

**Comment by jerone** [comment]
> I disabled `stats: 'errors-only'` to have webpack say exactly how long it takes for the build.

I'm currently using this is my `webpack.config.js`:
```js
	stats: {
		all: false,
		errors: true,
		moduleTrace: true,
		builtAt: true
	},
```

**Comment by bfred-it** [comment]
Webpack still shows a lot of useless information, instead of trying to filter it we decided to just use the SizePlugin

**Comment by jerone** [comment]
[**@bfred-it**](https://github.com/bfred-it) commented on [Jul 6, 2019, 6:19 AM GMT+2](https://github.com/sindresorhus/refined-github/pull/2199#issuecomment-508895743 "2019-07-06T04:19:06Z - Replied by Github Reply Comments"):
> Webpack still shows a lot of useless information, instead of trying to filter it we decided to just use the SizePlugin

Only thing missing from that is a timestamp, to know that the build actually happened. For example when the build fails, what errors belong to this build and which errors belong to the previous build. But maybe thats for another issue.

**Comment by bfred-it** [comment]
I added the `build-at` date as you suggested 👍 

I learned 2 things about the fork plugin:

- it already uses the `--incremental` flag
- it also shows the build time

It also shows some extra useless information that I can't disable though, so if you know how to do that, let me know:

```
No type errors found
Version: typescript 3.5.2
```

**Comment by jerone** [comment]
Great work 🤸‍♂️

**Pull request merged**